### PR TITLE
solves issue 1287 : Created structure of the doc using sphinx

### PR DESCRIPTION
**Pull request by TG1999**
I have used README.rst for making the doc. Solves #1287. Please review

### CODE REVIEW

**Comment by Abhishek-Dev09** [comment]
Thanks, in this way you have to search issue and solve.

**Comment by TG1999** [comment]
Screenshot 2019-05-08 at 12 56 33 AM
Here is a screenshot for the same

**Comment by TG1999** [comment]
Thanks, @Abhishek-Dev09 for guiding

**Comment by TG1999** [comment]
Hi @Abhishek-Dev09 @pombredanne can you suggest me how to resolve this. I have not changed these files in my commit so I don't know how it affected the code coverage .

**Comment by TG1999** [comment]
Now the code has passed all the checks, please review the changes

**Comment by pombredanne** [comment]
Thank you ++ 
Let me review this

**Comment by TG1999** [comment]
@pombredanne I have made the changes as per your review.

File: src/scancode/api.py [hunk 1]
```diff
@@ -228,6 +224,7 @@ def get_licenses(location, min_score=0, include_text=False,
 
             # FIXME: for sanity this should always be included?????
             if include_text:
+                matched_text = match.matched_text(whole_lines=False)
```
[the remark this answers is not in the document]
Comment (pombredanne) [reply src/scancode/api.py]: Can you remove that? Thanks!

**Comment by pombredanne** [comment]
@TG1999 also its looks like @DennisClark pushed the basic doc structure with this commit https://github.com/nexB/scancode-toolkit/commit/415d0c89267019b1c7bb35dc6bbb852d44419ec8 
Could you rebase and adapt/adopt @DennisClark structure?
Thanks!

**Comment by majurg** [comment]
@DennisClark Should this be moved to aboutcode? If so I will close and redirect the student to the proper location.

**Comment by majurg** [comment]
Changes incorporated by @AyanSinhaMahapatra via #27 and others. Closing this.

**Pull request closed without merging**